webpack/reducers/notifications.js

```diff
@@ -0,0 +1,12 @@
+import {
```

Comment:
It appears that this reducer is never added to the redux store. Based on the code as I read it, I don't see this doing anything at the moment. This should be removed for the moment.